crates/re_renderer/src/error_handling/error_tracker.rs

```diff
@@ -106,7 +106,11 @@ impl ErrorTracker {
             wgpu::Error::OutOfMemory { source: _ } => {
                 re_log::error!("A wgpu operation caused out-of-memory: {error}");
             }
-            wgpu::Error::Validation {
+            wgpu::Error::Internal {
+                source: _source,
+                description,
+            }
+            | wgpu::Error::Validation {
```

Comment:
Don't we care about the difference between these?

Reply:
Hmmmm I thought no, but I think now you're actually right. `Validation` means we messed up with the inputs we gave to wgpu/webgpu whereas `Internal` means that either the wgpu/webgpu messed up and/or a driver isn't behaving as expected